index.html

```diff
@@ -1102,6 +1102,11 @@ <h3><code>reason</code> member</h3>
       <div class="issue" data-number='117'></div>
     </section>
 
+    <section id="Internationalization">
+      <h2 id="x6-Internationalization">
```

Comment:
No need to add an id here, since we already have an id for the section.

Reply:
Thanks for the correction, has been updated to fix it